From: Hongxu Jia <hongxu.jia@windriver.com>

gtk.c: fix compiling failure with option -g -O

There was a compile failure with option -g -O
...
././gtk.c: In function 'main':
././gtk.c:2911:6: error: 'error' may be used uninitialized in this function [-Werror=maybe-uninitialized]
      fprintf(stderr, "%s: %s\n", pname, error);
      ^
cc1: all warnings being treated as errors
...

Fix by initializing pointer 'error' with NULL

Upstream-Status: Submitted [email discussion with upstream]

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Index: git/gtk.c
===================================================================
--- git.orig/gtk.c
+++ git/gtk.c
@@ -3578,7 +3578,7 @@ static void list_presets_from_menu(struc
 int main(int argc, char **argv)
 {
     char *pname = argv[0];
-    char *error;
+    char *error = NULL;
     int ngenerate = 0, px = 1, py = 1;
     bool print = false;
     bool time_generation = false, test_solve = false, list_presets = false;
